src/App.tsx

```
@@ -1,10 +1,16 @@
+
 const App = () => {
   return (
-    <div className="min-h-screen bg-white flex flex-col items-center justify-center">
-      <h1 className="text-5xl text-red-600 font-extrabold">핫 식스 프론트엔드 </h1>
-      <p className="mt-4 text-gray-500">Tailwind 테스트</p>
-    </div>
+    <MainLayout>
+      <AppRouter />
+    </MainLayout>
   );
+  return <RouterProvider router={router} />;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)